C/search/JumpSearch.c

```diff
@@ -0,0 +1,63 @@
+/*
+Jump Search is a searching algorithm for sorted arrays.
+The basic idea is to check fewer elements (than linear search) by jumping ahead by fixed steps or skipping some elements in place of searching all elements.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+
+int main()
+{
+	int n, ele;
+	printf("Enter the size of array");
+	scanf("%d", &n);
+	int a[n];
+	printf("\nEnter the sorted array elements");
+	for (int i = 0; i < n; i++)
+		scanf("%d", &a[i]);
+
+	printf("\nEnter the element to be searched");
+	scanf("%d", &ele);
+
+	int flag = -99;
+
+	//Instead of checking consecutive elements one by one, we jump elements by a distance of sqrt(n).
+	int jmp = sqrt(n);
```

Comment:
The ```main``` block should only have input-output statements. Make a new function and shift all the logic there.